src/utils.js

```diff
@@ -1,6 +1,9 @@
 'use strict'
 
 function idToHex (id) {
+  if (id == null) {
+    return
```

Comment:
Since this return value will be used, might be clearer to explicitly return `undefined`